platform/osgi/platform-osgi-configadmin/src/main/resources/OSGI-INF/permissions.perm

```diff
@@ -1,2 +1,63 @@
-(java.util.PropertyPermission "*" "read")
-(java.lang.RuntimePermission "createClassLoader")
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
✏️ Probably worth adding a comment to the effect that this file is not currently being used but is here for future deployments in which the OSGi security manager has been turned on.